bedrock/newsletter/templates/newsletter/knowledge-is-power.html

```diff
@@ -0,0 +1,54 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends 'base-protocol.html' %}
+
+{% block page_title %}{{ ftl('newsletters-subscribe-to-the-newsletter') }}{% endblock page_title %}
+
+{% block page_desc %}{{ ftl('newsletters-get-the-news-and-tips')}}{% endblock %}
+
+{% block body_class %}{{ super() }} newsletter-knowledge-is-power{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('protocol-newsletter') }}
+  {{ css_bundle('newsletter-knowledge-is-power') }}
+{% endblock %}
+
+{% block js %}
+  {{ js_bundle('newsletter') }}
+{% endblock %}
+
+{% block content %}
+<main role="main mzp-t-content-sm">
+  <section class="section-subscribe">
+    <header class="mzp-l-content mzp-t-content-lg header-container">
+      <div class="mozilla-products">
+        <ul class="mozilla-products-list">
+          <li>
+            <a href="{{ url('mozorg.home') }}" class="mzp-c-wordmark mzp-t-wordmark-md mzp-t-product-mozilla">Mozilla</a>
+          </li>
+          <li>
+            <a href="{{ url('firefox.browsers.index') }}" class="mzp-c-logo mzp-t-logo-md mzp-t-product-firefox">Firefox Browser</a>
+          </li>
+          <li>
+            <a href="{{ url('products.vpn.landing') }}" class="mzp-c-logo mzp-t-logo-md mzp-t-product-vpn">Mozilla VPN</a>
+          </li>
+          <li>
+            <a href="{{ url('firefox.pocket') }}" class="mzp-c-logo mzp-t-logo-md mzp-t-product-pocket">Pocket</a>
+          </li>
+        </ul>
+      </div>
+      <h1 class="page-title">{{ self.page_title() }}</h1>
+      <h2 class="section-title">{{ self.page_desc() }}</h2>
+    </header>
+    <div class="mzp-l-content mzp-t-content-sm">
+      {{ email_newsletter_form(newsletters='knowledge-is-power', include_language=True, include_title=False, spinner_color='#0c99d5') }}
```

Comment:
I'm not sure if we need a `spinner_color` for this newsletter, but I copied the one over from `newsletter/developer.html` just in case. 
I also made `include_language` True because there are multiple languages this newsletter serves based on the [JSON object](https://basket.mozilla.org/news/newsletters/). However, unsure if it should be `False` because the rest of the newsletters are set to False. Would appreciate feedback on this particular attribute.

Reply:
It looks like `include_language` defaults to `True`, so I'm not sure you need to include this here. It only seems to be the developer and contribute newsletters that don't include a language choice, and they both explicitly set `include_language=False`.